opencti-platform/opencti-front/src/private/components/analysis/opinions/OpinionCreation.tsx

```diff
@@ -137,7 +157,9 @@ export const OpinionCreationForm: FunctionComponent<OpinionFormProps> = ({
     externalReferences: [],
   };
 
-  const [commit] = useMutation(opinionCreationMutation);
+  const [commit] = userIsKnowledgeEditor
```

Comment:
Not a huge fan of that. Can you create two different forms depending on weither we have the capa or not ?

Reply:
Fixed